Development/sdp/sdp_grammar.cpp

```diff
@@ -96,6 +107,8 @@ namespace sdp
 
         const converter digits_converter{ jn2s, digits2jn };
 
+        const converter long_digits_converter{ jns2s, digits2jns };
```

Comment:
Very minor quibble... Long is often understood to refer to the `long` type... Maybe `big_digits_converter` as in "BigNum", something which is for holding numbers that are bigger than  the largest built-in type?